Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -38,6 +38,7 @@ protected function configure()
             ->setName('doctrine:fixtures:load')
             ->setDescription('Load data fixtures to your database')
             ->addOption('append', null, InputOption::VALUE_NONE, 'Append the data fixtures instead of deleting all data from the database first.')
+            ->addOption('groups', null, InputOption::VALUE_IS_ARRAY|InputOption::VALUE_OPTIONAL, 'Only fixtures with the following group attributes and the doctrine.fixtures.loader tag will be loaded')
```

Comment:
InputOption::VALUE_REQUIRED would be better than InputOption::VALUE_OPTIONAL here.

It does not make sense to specify a "groups" option to the command if it has no value.

Also, would it not be better to use singular "group" instead of "groups". As I understand it, if you want more than one group you need to repeat the option anyway.